topics/ecology/tutorials/regionalGAM/tutorial.md

```diff
@@ -318,8 +309,9 @@ This shows the abundance of *Pyronia tithonus*, over the weeks. We can see there
 >    - *"Plot title"*: add a meaningful title (e.g. `Pyronia tithonus phenology stacked visualization`)
 >    - *"Label for x axis"*: add a meaningful label (e.g. `Day number`)
 >    - *"Label for y axis"*: add a meaningful label <!-- TODO (e.g. ????? Ask Benjamin or Reto to know more)-->
->    - *"Type of plot"*: `Line only`
+>    - *"Type of plot"*: `Line only` (e.g. `nm value`)
```

Comment:
did you mean to put this on the line above (y-axis) or?

Reply:
oh no...  A mistake sorry!

Reply:
Copy / paste error, sorry. Corrected now!